### ISSUE TITLE
loading time

### ISSUE THREAD

**Issue by WeileiZeng** [sydrawat/covid19.in#1]
The loading of the numbers took 2-3 seconds in US. 

I assume you read the numbers from API at each time. To make it faster, you can use Github action or other automation to fetch real-time data every 10 mins, and put the numbers into static files.

**Comment by sydrawat** [comment]
@WeileiZeng Yeah the API call renders are slow. I was thinking of  either fetching the data from the frontend and update the server cache from there. Or, use a server as a proxy. Then we would fetch the data from that server. The server could first check the cache and only send a request to the API if the cache is empty. A simple Cron job that updates the API every X minutes so the data is always fresh should do the job!

**Comment by WeileiZeng** [comment]
Yeah, that works as well. I think usually people only check the information about once a day. So a delay in several minutes would have no effect at all.
